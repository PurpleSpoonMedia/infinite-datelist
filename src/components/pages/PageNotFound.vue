<template>
  <div>
    <h1 class="text-center">404 error! Page not found</h1>
  </div>
</template>
