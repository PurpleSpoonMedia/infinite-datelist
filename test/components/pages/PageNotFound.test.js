import { shallowMount } from '@vue/test-utils';
import PageNotFound from '../../../src/components/pages/PageNotFound.vue';

describe('PageNotFound', () => {
  test('render PageNotFound correctly', () => {
    const wrapper = shallowMount(PageNotFound);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
