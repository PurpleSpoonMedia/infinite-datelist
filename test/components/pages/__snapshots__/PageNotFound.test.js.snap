// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound render PageNotFound correctly 1`] = `
<div>
  <h1 class="text-center">404 error! Page not found</h1>
</div>
`;
